index.js

```diff
@@ -167,6 +162,34 @@ module.exports = {
       },
     ],
     "decorator-position/decorator-position": ["error", { printWidth: 80 }],
+    "simple-import-sort/imports": [
+      "error",
+      {
+        groups: [
+          [
+            // Ember/glimmer
+            "^@ember/",
+            "^@glimmer/",
```

Comment:
```suggestion
            "^@glimmer/",
            "^@ember/",
```

so that `@glimmer/component` can be first, then any of the `@ember` stuff